include/onnxruntime/core/graph/graph.h

```diff
@@ -753,7 +753,6 @@ class Graph {  // NOLINT(clang-analyzer-optin.performance.Padding): preserve exi
   cannot be overridden at runtime. If the initializer is not found or is not constant, a nullptr is returned.
   @param check_outer_scope If true and the graph is a subgraph,
          check ancestor graph/s for 'name' if not found in 'graph'.
-  @remarks check_outer_scope of true is not supported in a minimal build
```

Comment:
This has not been a limitation since the logic was changed a long long time ago to use implicit inputs for determining outer scope values. 

Originally the implementation used info in the resolve context, which was only populated during Graph::Resolve, which doesn't happen in a minimal build. Using implicit inputs is way cleaner and works in all builds.